django_switchuser/urls.py

```diff
@@ -2,7 +2,12 @@
 
 from . import views
 
-urlpatterns = patterns("",
+patterns = [
     url(r"^$", views.su_login, name="su-login"),
     url(r"^logout$", views.su_logout, name="su-logout"),
-)
+]
+
+try:
+    urlpatterns = patterns("", *patterns)
+except NameError:
```

Comment:
1. The `patterns = […]` clobbers the `from … import patterns`
2. Why would a `NameError` be raised? (this isn't something I've seen before)